prometheus-collectors/class-potential-multi-dataset-queries-collector.php

```diff
@@ -49,6 +50,21 @@ public function sql_query_log( $query ): void {
 					$blog_ids_count >= 3 ? '3+' : (string) $blog_ids_count,
 				]
 			);
+
+			$backtrace = function_exists( 'wp_debug_backtrace_summary' ) ? wp_debug_backtrace_summary( null, 4, false ) : []; // phpcs:ignore
+			\Automattic\VIP\Logstash\log2logstash(
+				[
+					'severity' => 'debug',
+					'feature'  => 'potential_multi_dataset_queries',
+					'message'  => 'Potential multi dataset query detected',
+					'blog_id'  => get_current_blog_id(),
+					'extra'    => [
+						'uri'       => isset( $_SERVER['REQUEST_URI'] ) ? esc_url_raw( $_SERVER['REQUEST_URI'] ) : '',
+						'method'    => isset( $_SERVER['REQUEST_METHOD'] ) ? sanitize_text_field( $_SERVER['REQUEST_METHOD'] ) : '',
+						'backtrace' => $backtrace,
```

Comment:
It may be helpful to include `$last_global_table`, `$last_blog_table` and `blog_ids_count`.

Reply:
Ohh good idea!